ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -29,13 +29,29 @@
 from ironic.drivers.modules.drac import common as drac_common
 from ironic.drivers.modules.drac import job as drac_job
 from ironic.drivers.modules.drac import management as drac_mgmt
+from ironic.drivers.modules.redfish import utils as redfish_utils
+from ironic.drivers.modules.drac import utils as drac_utils
+from ironic.tests.unit.db import utils as db_utils
 from ironic.tests.unit.drivers.modules.drac import utils as test_utils
 from ironic.tests.unit.objects import utils as obj_utils
 
+sushy = importutils.try_import('sushy')
+
 dracclient_exceptions = importutils.try_import('dracclient.exceptions')
 
 INFO_DICT = test_utils.INFO_DICT
 
+DRAC_REDFISH_INFO_DICT = db_utils.get_test_redfish_info()
+
+_SERVICE_ROOT = '/redfish/v1/Managers/'
+
+authenticator = sushy.auth.BasicAuth(
+                    DRAC_REDFISH_INFO_DICT["redfish_username"], 
+                    DRAC_REDFISH_INFO_DICT["redfish_password"])
+url = '%s%s' % (DRAC_REDFISH_INFO_DICT["redfish_address"], _SERVICE_ROOT)
+conn = sushy.Sushy(url, verify=False, auth=authenticator)
+manager = conn.get_manager('iDRAC.Embedded.1')
+oem_manager = manager.get_oem_extension('Dell')
```

Comment:
Please sync up with Mahendra on how we're going to mock the sushy library.  See Julia's and my comments here: https://review.opendev.org/#/c/693328/7/ironic/tests/unit/drivers/modules/drac/test_inspect.py

Reply:
Thanks Chris, sure I will sync up with Mahendra and will address this.